src/openrct2/world/Climate.cpp

```diff
@@ -123,6 +122,8 @@ void ClimateUpdate()
 {
     PROFILED_FUNCTION();
 
+    auto& gameState = GetGameState();
```

Comment:
This can be moved down to just before itsefursr use.

Reply:
It's actually fine here, there are more variables using the gameState like CurrentTicks so it would have to be moved here again eventually.